p2panda-rs/src/hash/hash.rs

```diff
@@ -10,8 +10,11 @@ use yasmf_hash::{YasmfHash, BLAKE3_HASH_SIZE, MAX_YAMF_HASH_SIZE};
 use crate::hash::HashError;
 use crate::Validate;
 
+/// This is the size of p2panda entries' hashes
+pub const HASH_SIZE: usize = BLAKE3_HASH_SIZE;
```

Comment:
Should we publish `HASH_SIZE` to let library consumers gauge the likelihood of collisions or is this an internal that they don't need to know about?

Reply:
Similar question for `SIGNATURE_SIZE`.

Reply:
Blake3 claims to be resistent to collision attacks, so hopefully that will not happen :-D also signatures will not collide, otherwise ed25519 would be badly broken.

Reply:
On another note: I really don't like this API where we have to keep track of the hash size of something which should be as generic as `yamf-hash` .. but I guess thats more an API issue with the crate than with us.

Reply:
Yes, I agree. I will un-publish the constants but I would rather have constants than just lots of `32` and `64` all over the code 😅